packages/cryptobox/test/CryptoboxSpec.common.js

```diff
@@ -175,21 +177,37 @@ describe('cryptobox.Cryptobox', () => {
     });
   });
 
+  describe('PreKeys', () => {
+    describe('"serialize_prekey"', () => {
+      it('generates a JSON format', async done => {
+        const box = new cryptobox.Cryptobox(store, 10);
+        box.identity = await Proteus.keys.IdentityKeyPair.new();
+        const preKeyId = 72;
+        const preKey = await Proteus.keys.PreKey.new(preKeyId);
+        const json = box.serialize_prekey(preKey);
+        expect(json.id).toBe(preKeyId);
+        const decodedPreKeyBundleBuffer = sodium.from_base64(json.key, sodium.base64_variants.ORIGINAL).buffer;
+        expect(decodedPreKeyBundleBuffer).toBeDefined();
+        done();
+      });
+    });
+  });
+
   describe('Sessions', () => {
     let box = undefined;
     const sessionIdUnique = 'unique_identifier';
 
-    beforeEach(done => {
+    beforeEach(async done => {
```

Comment:
No need for `async` here

Reply:
But `Proteus.keys.IdentityKeyPair.new` is async.

Reply:
But you have `async` here: https://github.com/wireapp/wire-web-packages/pull/272/files#diff-f3493822b0bf4f72fb4ebffff1c3e2acR203